src/Css.elm

```diff
@@ -2324,6 +2404,16 @@ prop3 key argA argB argC =
 prop4 : String -> Value a -> Value b -> Value c -> Value d -> Mixin
 prop4 key argA argB argC argD =
     property key (String.join " " [ argA.value, argB.value, argC.value, argD.value ])
+    
+    
+{-| Sets ['float'](https://developer.mozilla.org/en-US/docs/Web/CSS/float)
+floatAlign : FloatAlign compatible -> Mixin
+
+    floatAlign right
+-}
+floatAlign : (ExplicitLength IncompatibleUnits -> Mixin) -> Mixin
+floatAlign fn =
+    getOverloadedProperty "floatAlign" "float" (fn lengthForOverloadedProperty)
```

Comment:
@colvardrm I changed `float` to `num` on `master` so you're all set to rename! Also I split the exports line into multiple lines so it should be easier to avoid merge conflicts this time. 😅